src/main/java/model/ConnectionPool.java

```diff
@@ -4,39 +4,44 @@
 import java.sql.DriverManager;
 import java.sql.SQLException;
 import java.util.ResourceBundle;
-import java.util.concurrent.ArrayBlockingQueue;
 import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.LinkedBlockingQueue;
 
 public class ConnectionPool {
-    private static volatile ConnectionPool uniqueInstance;
-    private BlockingQueue<Connection> availableConnections = new ArrayBlockingQueue<Connection>(25);
-
-    private ConnectionPool(int connectionsCount) {
-        for (int i = 0; i < connectionsCount; i++) {
-            availableConnections.add(createConnect());
+    private static volatile ConnectionPool instance;
+    private BlockingQueue<Connection> availableConnections = new LinkedBlockingQueue<Connection>();
+    private ResourceBundle resource = ResourceBundle.getBundle("resources/properties/database/database");
```

Comment:
1. Create a separate class like Properties, 
2. Move ResourceBundle to that class property.
3. Create a public static method getProperty(String name), that will return value from resourceBundle object.